components/Airdrop/StepSubmitAddress/StepSubmitAddress.tsx

```diff
@@ -52,18 +58,51 @@ export default function StepSubmitAddress({ onNext }: Props) {
             <TextField {...confirmPubAddress} className="max-w-full" />
           )}
         </div>
+        <div
+          style={{
+            marginTop: '1em',
+            padding: '.75em',
+            backgroundColor: '#eeeeee',
+          }}
+        >
+          <div>
+            IMPORTANT: you must have access to this address to receive your
+            airdrop. We will not be able to recover lost tokens sent to
+            inaccessible accounts. We strongly suggest{' '}
+            <a
+              className="underline"
+              target="_blank"
+              href="https://ironfish.network/docs/onboarding/iron-fish-wallet-commands#export-an-account"
+              rel="noreferrer"
+            >
+              exporting your wallet information
+            </a>{' '}
+            to ensure you always have access.
+          </div>
+          <div style={{ marginTop: '1em' }}>
+            <input
+              type="checkbox"
+              onClick={() => {
+                setButtonDisabled(!buttonDisabled)
+              }}
+            ></input>{' '}
```

Comment:
nit: This works, but it's indirect since we're relying on the fact that a checkbox is unchecked by default.

A more declarative way to do would be:

```js
const [isChecked, setIsChecked] = useState(false);
...
<input
  type="checkbox"
  checked={isChecked}
  onChange={e => {
    setIsChecked(e.target.checked)
  }}
/>
...
<button disabled={!isChecked}>hi</button>
```

Feel free to update it or keep it as is.